devicetypes/smartthings/zwave-device-multichannel.src/zwave-device-multichannel.groovy

```diff
@@ -435,3 +434,9 @@ private encap(cmd, endpoint) {
 private encapWithDelay(commands, endpoint, delay=200) {
 	delayBetween(commands.collect{ encap(it, endpoint) }, delay)
 }
+
+def updated() {
+    childDevices.each {
+        if (it.device.isComponent) { it.save([isComponent: false, componentLabel: null, componentName: null]) }
+    }
+}
```

Comment:
@tpmanley happy news is that I have confirmed that this works as we expect.

Reply:
So I can probably use this for the Inovelli device, then.